core/zms/src/main/rdl/Template.tdl

```diff
@@ -7,11 +7,22 @@ include "Role.tdl";
 include "Policy.tdl";
 include "ServiceIdentity.tdl"
 
+//MetaData for template.
+type MetaData Struct {
```

Comment:
let's call this something more specific like TemplateMetaData. Since this ends up as a Class name, MetaData is too generic

Reply:
Done.